src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -1167,7 +1167,12 @@ public void Terminate()
             ScheduleStdOutProcessing(@"*stopped,reason=""exited"",exit-code=""42""");
         }
 
-        public void Detach() { }
+        public void Detach()
+        {
+            // Fake stopped message to send ProgramDestroyEvent.
+            ScheduleStdOutProcessing(@"*stopped,reason=""disconnected""");
```

Comment:
I don't see the reason `"disconnected"` being handled on the `OnStopped` handler (in Debugger.cs).

Shouldn't it cause the debugger to exit (`-gdb-exit`)?